environments/production/app-processes.yml

```diff
@@ -152,14 +152,14 @@ celery_processes:
       concurrency: 4
       max_tasks_per_child: 5
   celery4:
-    celery:
+    celery,case_import_queue:
       concurrency: 6
       max_tasks_per_child: 5
     export_download_queue:
       concurrency: 6
       max_tasks_per_child: 5
   hqcelery4.internal-va.commcarehq.org:
-    celery:
+    celery,case_import_queue:
```

Comment:
Head up: This change will conflict with this PR: https://github.com/dimagi/commcare-cloud/pull/2342. (In general we're removing the rackspace celery machines)